documentation/book/proc-deploying-kafka-cluster-openshift.adoc

```diff
@@ -28,5 +30,5 @@ oc apply -f examples/kafka/kafka-persistent.yaml
 ----
 
 .Additional resources
-* For more information about deploying the Cluster Operator, see xref:cluster-operator-str[]
+* For more information about deploying the Cluster Operator, see xref:cluster-operator-{context}[]
```

Comment:
Replace {context} with 'str' which will be the default.

Reply:
This got skipped

Reply:
Same comment, I can fix it if you want.